### ISSUE TITLE
Scale columns acording to their contents

### ISSUE THREAD

**Issue by shrx** [topisani/Discurses#6]
See screenshot:
![screenshot_2016-10-06_13-58-05](https://cloud.githubusercontent.com/assets/810337/19151453/08efbf84-8bcd-11e6-8615-afd60345593a.png)

Possible fix: truncate usernames to fixed max length.

**Comment by topisani** [comment]
yup i am aware, i tried to be intelligent about it, and apparently i tried to be too intelligent about it ;) I'll take a look at it. Thanks for the report, keep it up!

**Comment by topisani** [comment]
Columns need to scale acording to their contents
